### DISCUSSION TITLE
Multiple viewport meta tag on Google Rich Text Result test

### DISCUSSION THREAD

**Discussion by ghost** [vercel/next.js#33822] [Help]
Hi guys,
I am facing a very weird issue where if I check inspect element or view source everything seems to be fine but if I test my page on Richtext Result test of the google then it's showing 2 viewport meta tags. The funny thing is I haven't written my own viewport meta tag anywhere in the code as I know NextJS auto adds that meta tag. 

Here are screenshot relevant to the issues:
Inspect Element
![image](https://user-images.githubusercontent.com/91734603/151742721-49e0c631-9a3d-4e39-aacd-ac244bd12065.png)

View Source
![image](https://user-images.githubusercontent.com/91734603/151742776-66646dfd-ccca-4376-abc8-52e4f170717e.png)

Google Rich Results Test
![image](https://user-images.githubusercontent.com/91734603/151743011-3c87818d-e7aa-4672-a2ce-3e9ebff4224b.png)
![image](https://user-images.githubusercontent.com/91734603/151743038-6027507a-dc34-4cfe-a5f5-178fdeca3aca.png)

PS:
I am using NextJS version 11.1.2

If anyone has faced a similar issue, Kindly let me know what should be done.

**Comment by krish-penumarty** [comment]
I'm currently facing the same issue! I have a custom _document.js file that adds <Head /> from next/document, but also use next/head value in individual pages to add title and meta tags. Were you able to find a solution?
